buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/DistributionTest.kt

```diff
@@ -129,7 +129,7 @@ abstract class DistributionTest : Test() {
 class LocalRepositoryEnvironmentProvider(project: Project) : CommandLineArgumentProvider, Named {
 
     @Internal
-    val localRepo = project.files()
```

Comment:
:o: What's this `files` method? It's not on `Project` so I guess it's a DSL extension? What's the difference?

Reply:
`files()` gives you a ConfgurableFIleCollections. Usually you wound do `file("someFile.txt")` in a build.

But I don't understand what the issue is here, as this was removed and replaced with the better `objects.fileCollection()` to make clear we really only need to create an empty file collection here to instantiate the property. :)

Reply:
No issue, I was wondering if this `files()` method, because it's an extension method, could be changed to call `objects.fileCollection()` instead, so that you don't have to change in all places.